### PR TITLE
API Release

### PR DESCRIPTION
**Pull request by bkb2135**

### CODE REVIEW

File: prompting/api/gpt_endpoints/api.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    #     model=body.get("model"),
+    #     seed=body.get("seed"),
+    #     response=response_event,
+    # )
+
+    # task_scorer.add_to_queue(
+    #     task=task, response=response_event, dataset_entry=task.dataset_entry, block=-1, step=-1, task_id=task.task_id
+    # )
+
+    # return response
```
Comment (dbobrenko) [inline prompting/api/gpt_endpoints/api.py:172]:
```suggestion
```

File: api_keys.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"0566dbe21ee33bba9419549716cd6f1f": {"rate_limit": 10, "usage": 0}, "e03da67439c0b7e7a622dde4fa3cf857": {"rate_limit": 10, "usage": 0}}
```
Comment (dbobrenko) [inline api_keys.json:1]: Make sure to remove this
```suggestion
{}
```

**Review by dbobrenko — CHANGES_REQUESTED** [review]
Make sure to remove API keys from the PR

**Review by Hollyqui — APPROVED** [review]
LGTM

**Pull request merged**